baremaps-cli/src/main/java/com/baremaps/cli/OpenApi.java

```diff
@@ -71,13 +79,24 @@ public Integer call() throws Exception {
     mapper.setSerializationInclusion(JsonInclude.Include.NON_NULL);
     jdbi.getConfig(Jackson2Config.class).setMapper(mapper);
 
+    BeanConfig beanConfig = new BeanConfig();
+    beanConfig.setVersion("1.0.0");
+    beanConfig.setSchemes(new String[]{"http"});
+    beanConfig.setHost("localhost:9000");
```

Comment:
Since we already have the port in the config ?

```suggestion
    beanConfig.setHost(String.format("localhost:%s", port));
```

Reply:
Ideally, we should get the host from the HTTP header of the request. would it make sense to inject the value of the Host header?

Reply:
![image](https://user-images.githubusercontent.com/13271765/130062151-f3b540a8-b49a-482e-9ae3-07a1c7918e5e.png)

Unfortunately, it requires a bit of logic. In case of "X-Forwareded-For" header and stuff like that. I thought JAX-RS had an abstraction for that. I have to test to be sure.

Reply:
@danduk82 will create an abstraction for this for another use case (openapi). I think we'll be able to reuse it.

Reply:
In fact it looks like Jersey provides a mean to get the host through UriInfo.getBaseUri (14c8d08908). @danduk82 can you confirm that this approach is working? if so, I will adapt the edit and serve command accordingly.

Reply:
It depends on the reverse proxy you are using :-/ 
Some place does not replace the `Host` some does. I think we can start with `UriInfo.getBaseUri` and elaborate further if it fail largely (but I don't think so).

Reply:
This we addressed in #340